zetaclient/zetabridge/zetacore_bridge.go

```diff
@@ -248,17 +247,23 @@ func (b *ZetaCoreBridge) UpdateZetaCoreContext(coreContext *corecontext.ZetaCore
 	keyGen, err := b.GetKeyGen()
 	if err != nil {
 		b.logger.Info().Msg("Unable to fetch keygen from zetabridge")
+		return err
 	}
 
-	tssPubKey := ""
 	tss, err := b.GetCurrentTss()
 	if err != nil {
 		b.logger.Debug().Err(err).Msg("Unable to fetch TSS from zetabridge")
-	} else {
-		tssPubKey = tss.GetTssPubkey()
+		return err
```

Comment:
Can we print above log a bit louder (with info)? We had a GetTss issue and hotfix early in mainnet and we saw no errors when it happened.